tutor/test_new_t1_14_TeacherReading.py

```diff
@@ -0,0 +1,2052 @@
+TESTS = os.getenv(
+    'CASELIST',
+    str([
+        132519, 132520, 132521, 132522, 132523,
+        132560, 132524, 132561, 132525, 132528,
+        132527, 132526, 132530, 132531, 132529,
+        132532, 132533, 132534, 132577
+
+    ])
+)
+
+
+@PastaDecorator.on_platforms(BROWSERS)
+class TestCreateAReading(unittest.TestCase):
+    """T1.14 - Create a Reading."""
+
+    def setUp(self):
+        """Pretest settings."""
+        self.ps = PastaSauce()
+        self.desired_capabilities['name'] = self.id()
+        if not LOCAL_RUN:
+            self.teacher = Teacher(
+                use_env_vars=True,
+                pasta_user=self.ps,
+                capabilities=self.desired_capabilities
+            )
+        else:
+            self.teacher = Teacher(
+                use_env_vars=True
+            )
+        self.teacher.login()
+        self.teacher.select_course(appearance='college_physics')
+
+    def tearDown(self):
+        """Test destructor."""
+        if not LOCAL_RUN:
+            self.ps.update_job(
+                job_id=str(self.teacher.driver.session_id),
+                **self.ps.test_updates
+            )
+        try:
+            self.teacher.delete()
+        except:
+            pass
+
+    # Case C132519 001 - Teacher | Create and publish a new open reading for all periods
+    @pytest.mark.skipif(str(132519) not in TESTS, reason='Excluded')
+    def test_teacher_create_and_publish_a_new_open_reading_132519(self):
+        """Add a reading using the Add Assignment drop down menu.
+        #Steps
+        Click on the 'Add Assignment' button
+        Click on the 'Add Reading' option
+        ***The teacher is taken to the Add Reading Assignment page***
+
+        Enter an assignment name into the Assignment name text box [user decision]
+        ***Typed text appears in the input box***
+
+        Enter an assignment description into the Assignment description or special instructions text box
+        ***Typed text appears in input box***
+
+        ['All Sections' should be selected by default]
+        Click on the Open Date text field and click on today's date on calendar pop-up
+        Click on the Open Time text field and enter 12:00am.
+        Click on the Due Date text field and click on desired due date on calendar pop-up
+        Click on the "+ Add Readings" button
+        Click on section(s) to add to assignment
+        Scroll to bottom of the page
+        Click on the "Add Readings" button
+        Click on the "Publish" button
+
+        # Expected result
+        ***Takes user back to calendar dashboard. An opened assignment appears on user calendar dashboard on due date with correct readings***
+
+        """
+        self.ps.test_updates['name'] = 't1.14.001' \
+            + inspect.currentframe().f_code.co_name[4:]
+        self.ps.test_updates['tags'] = ['t1', 't1.14', 't1.14.001', '7992']
+        self.ps.test_updates['passed'] = False
```

Comment:
the name and tags should be updated to match the new case ID, epic name and product ('tutor')